app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditItemAdapter.java

```diff
@@ -27,8 +30,11 @@ public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
 
     @Override
     public void onBindViewHolder(ViewHolder holder, int position) {
-        holder.itemView.setTag(tagsList.get(position));
-        holder.editItemViewModel.setModel(tagsList.get(position));
+        final Tag tag = tagsList.get(position);
+
+        holder.itemView.setTag(tag);
+        holder.editItemViewModel.setModel(tag);
+        holder.editItemViewBinding.eraseButton.setOnClickListener(v -> onTagActionListener.onClear(tag));
```

Comment:
I would like to see TODO comment, smth like "think about moving it to onCreateViewHolder in the future"